# Generated by Django 5.1.6 on 2025-04-24 15:42

from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('catalog', '0037_alter_setlistitem_track'),
    ]

    operations = [
        migrations.CreateModel(
            name='Track404',
            fields=[
                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('beatport_track_id', models.BigIntegerField(verbose_name='Beatport Track ID')),
            ],
            options={
                'constraints': [models.UniqueConstraint(fields=('beatport_track_id',), name='beatport_track_id_unique', violation_error_message='This track ID from Beatport is already marked as 404.')],
            },
        ),
    ]
